packages/platform-android/native_modules.gradle

```diff
@@ -265,6 +275,15 @@ class ReactNativeModules {
         if (!androidConfig["buildTypes"].isEmpty()) {
           reactNativeModulesBuildVariants.put(nameCleansed, androidConfig["buildTypes"])
         }
+        if(androidConfig.containsKey("customDependencyConfiguration")) {
+          reactNativeModuleConfig.put("customDependencyConfiguration", androidConfig["customDependencyConfiguration"])
+        } else if (project.containsKey("customDependencyConfiguration")) {
```

Comment:
Shouldn't it have priority over `androidConfig`?

Reply:
We've agreed that config per module should have higher priority :)

Reply:
Yeah, I thought `project` relates to the module. LGTM